apps/prairielearn/src/pages/administratorQuery/administratorQuery.ts

```diff
@@ -82,33 +114,36 @@ router.post(
       encoding: 'utf8',
     });
 
-    const queryParams = _.pick(
-      req.body,
-      _.map(info.params || {}, (p) => p.name),
-    );
-    const params = {
+    const queryParams = {};
+    if (info.params) {
+      info.params.forEach((p) => {
+        queryParams[p.name] = req.body[p.name];
+      });
+    }
+
+    const params: AdministratorQueryRunParams = {
       name: req.params.query,
       sql: querySql,
       params: JSON.stringify(queryParams),
       authn_user_id: res.locals.authn_user.user_id,
       error: null,
       result: null,
+      formatted_date: null,
```

Comment:
Why is `formatted_date` here? AFAICT it's not used in either of the below queries.

Reply:
Resolved in commit 12dc009.